### ISSUE TITLE
Update error

### ISSUE THREAD

**Issue by DxrkPxwer** [JordanSantiagoYT/FNF-JS-Engine#717]
### Describe your bug here. If you are modding, try testing your bug in a clean version of the engine instead. Also, if you're using an older version of JSE, please try the latest version/action build. Also, be sure to check the pinned 'JS Engine: Known Issues' issue, to check if your issue hasn't already been found!

I've recently found out about the new JS engine hotfix (1.37.1) and i've decided to update it using the in-game updater
however it crashed right in the end
Below, the crash log

### Command Prompt/Terminal/Crash logs (if existing)

Uncaught Error: Custom([file_write,./update/raw//assets/songs/thorns/Inst.ogg])
C:\hostedtoolcache\windows\haxe\4.3.4\x64\std/cpp/_std/sys/io/FileOutput.writeBytes() [line 44]
C:\hostedtoolcache\windows\haxe\4.3.4\x64\std/haxe/io/Output.write() [line 107]
JSEZip.unzip() [line 50]
UpdateState.onDownloadComplete() [line 313]
Please report this error to the GitHub page: https://github.com/JordanSantiagoYT/FNF-JS-Engine
The engine has saved a crash log inside the crash folder, If you're making a GitHub issue you might want to send that!

### Have you identified any steps to reproduce the bug? If so, please describe them below in as much detail as possible. Use images if possible.

Basically open up the update screen and wait, then kaboom

### Are you modding a build from source or with Lua?

Source

### What is your build target?

Windows

### Did you edit anything in this build? If so, mention or summarize your changes.

Bruh i've just tried to update it

### If you use 1.12.0 or earlier, did you have Optimized Chart Loading turned on?

None

### Did you check for any similar issues to what you're reporting? Check, then come back here. If there is a similar issue, then do not report the issue, otherwise it will be marked as a duplicate.

Yes

**Comment by JordanSantiagoYT** [comment]
yeah this isnt something i can really fix

**Comment by DxrkPxwer** [comment]
why tho

> yeah this isnt something i can really fix


cuz like, last time when you advised that you couldn't update with the ingame updater, it worked for me

**Comment by DxrkPxwer** [comment]
what 
i've just updated it and it went normal?
smths werid as hell

**Comment by DxrkPxwer** [comment]
> what i've just updated it and it went normal? smths werid as hell

aaaand it didn't update
what

**Comment by StinkTheStinker** [comment]
just do it manualy

**Comment by DxrkPxwer** [comment]
> just do it manualy

yea i found out the reason (or atleast that's what i thought)
the source name thingy changed???? maybe thats the cause

**Comment by moxie-coder** [comment]
> > just do it manualy
> 
> yea i found out the reason (or atleast that's what i thought) the source name thingy changed???? maybe thats the cause

![image](https://github.com/user-attachments/assets/33ea1835-7545-4fab-918e-820a76f5183e)

it was said right here
